CrossRef.js

```diff
@@ -410,14 +307,52 @@ var testCases = [
 				"bookTitle": "Heimat, Region, and Empire",
 				"ISBN": "9780230391116",
 				"publisher": "Palgrave Macmillan",
-				"language": "en",
 				"date": "2012-10-17",
 				"DOI": "10.1017/CCOL0521858429.016",
```

Comment:
something is off here - are we not starting with an empty item? This is the Hitchens DOI, not the one belonging to this chapter.